### PR TITLE
rebase patch

### PR DESCRIPTION
**Pull request by MaximilianMeister**
Signed-off-by: Maximilian Meister <mmeister@suse.de>

### CODE REVIEW

**Comment by MaximilianMeister** [comment]
probably introduced by https://github.com/kubic-project/velum/pull/650

File: packaging/suse/patches/0_set_default_salt_events_alter_time_column_value.rpm.patch [hunk 1]
```diff
@@ -1,8 +1,8 @@
 diff --git a/db/schema.rb b/db/schema.rb
-index d37f481..ec3219f 100644
+index ce48cc5..d2b2b16 100644
```
Comment (bear454) [inline packaging/suse/patches/0_set_default_salt_events_alter_time_column_value.rpm.patch]: I have no idea how this is derived.

File: packaging/suse/patches/0_set_default_salt_events_alter_time_column_value.rpm.patch [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
 --- a/db/schema.rb
 +++ b/db/schema.rb
-@@ -107,7 +107,7 @@ ActiveRecord::Schema.define(version: 20181708070234) do
+@@ -133,7 +133,7 @@ ActiveRecord::Schema.define(version: 20181715075511) do
```
Comment (bear454) [inline packaging/suse/patches/0_set_default_salt_events_alter_time_column_value.rpm.patch]: This is exactly the thing I was asking about in IRC...

**Comment by jordimassaguerpla** [comment]
You need to update the patch file. All you need to do is create a patch that, when applied to the db/schema.rb, it does modification:

-    t.datetime "alter_time",                    null: false
+    t.column   "alter_time",   "DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP"

You only have to do this if you are adding a new database migration.

**Comment by jordimassaguerpla** [comment]
@bear454 Didn't we already discussed that :) ?

https://github.com/kubic-project/velum/pull/515#issuecomment-387800205

Feel free to create a PR to fix it :)

**Pull request merged**